pkg/backup/schedule_exec.go

```diff
@@ -574,10 +581,16 @@ func init() {
 					ExecutorPTSMetrics: &pm,
 					RpoMetric: metric.NewGauge(metric.Metadata{
 						Name:        "schedules.BACKUP.last-completed-time",
-						Help:        "The unix timestamp of the most recently completed backup by a schedule specified as maintaining this metric",
+						Help:        "The unix timestamp of the most recently completed tenant backup by a host schedule specified as maintaining this metric",
```

Comment:
I would revert this change. The metric is bumped by a schedule within the cluster, so "host" is misleading. We also typically refer to backups taken _within_ a tenant as just a "backup". A "tenant backup" typically refers to a backup _of a_ tenant.